src/user/views/author_views.py

```diff
@@ -725,6 +732,8 @@ def publications(self, request, pk=None):
 
         serializer_data = serializer.data
 
+        cache.set(cache_key, serializer_data, timeout=3600)
+
         return self.get_paginated_response(serializer_data)
```

Comment:
I wonder if we should cache the paginated response instead. Without the paginated response there will be no pagination if returned from cache

Reply:
Thanks for pointing it out. I believe we shouldn't cache paginated results like this. Instead, I changed the code to cache the retrieved documents and wrap it in the pagination object accordingly.